docs/bigquery-usage.rst

```diff
@@ -1,9 +1,5 @@
-.. toctree::
-  :maxdepth: 0
-  :hidden:
```

Comment:
Should be fine to drop it.  Maybe if we had a landing page for each API, it would make sense there.